homeassistant/components/supla/__init__.py

```diff
@@ -30,7 +30,10 @@
 SUPLA_FUNCTION_HA_CMP_MAP = {
     "CONTROLLINGTHEROLLERSHUTTER": Platform.COVER,
     "CONTROLLINGTHEGATE": Platform.COVER,
+    "CONTROLLINGTHEGARAGEDOOR": Platform.COVER,
```

Comment:
You should split this PR:
- one PR for adding an extra cover type to existing cover platform.
- one PR for adding a brand new light platform.